acme-cli/build.gradle

```diff
@@ -1,36 +1,28 @@
 plugins {
     id "groovy"
-    id "net.ltgt.apt-eclipse" version "0.21"
-    id "com.github.johnrengelman.shadow" version "5.0.0"
+    id "com.github.johnrengelman.shadow"
     id "application"
 }
 
-
-
-version "0.1"
-group "acme.cli"
-
-repositories {
-    mavenCentral()
-    maven { url "https://jcenter.bintray.com" }
-}
-
 configurations {
     // for dependencies that are needed for development only
-    developmentOnly 
+    developmentOnly
 }
 
 dependencies {
     annotationProcessor platform("io.micronaut:micronaut-bom:$micronautVersion")
     annotationProcessor "io.micronaut.configuration:micronaut-picocli"
     annotationProcessor "io.micronaut:micronaut-inject-java"
     annotationProcessor "io.micronaut:micronaut-validation"
-    implementation platform("io.micronaut:micronaut-bom:$micronautVersion")
     implementation "io.micronaut:micronaut-runtime"
     implementation "info.picocli:picocli"
     implementation "io.micronaut.configuration:micronaut-picocli"
     implementation "io.micronaut:micronaut-inject"
     implementation "io.micronaut:micronaut-validation"
+
+    implementation group: 'org.shredzone.acme4j', name: 'acme4j-client', version: '2.6'
+    implementation group: 'org.shredzone.acme4j', name: 'acme4j-utils', version: '2.6'
+
     runtimeOnly "ch.qos.logback:logback-classic:1.2.3"
     testAnnotationProcessor platform("io.micronaut:micronaut-bom:$micronautVersion")
     testAnnotationProcessor "io.micronaut:micronaut-inject-java"
```

Comment:
Are the duplicate `testAnnotationProcessor`'s necessary?
If so that's kind of obnoxious

Reply:
Nope....cleaned up here 511bda5